vocab/credentials/v2/credentials.jsonld

```diff
@@ -0,0 +1,385 @@
+{
+    "@context": [
+        {
+            "@version": 1.1
```

Comment:
We've been deliberately avoiding setting `@version` everywhere -- JSON-LD 1.1 parsers handle this automatically via feature detection.

Reply:
Ah. That was a leftover (I presume) from the time we thought the `@version` was required in JSON-LD 1.1. (If you remember, this change came pretty much at the end of the JSON-LD 1.1 development.) Thanks.

Done.